src/test/groovy/com/github/vkorobkov/jfixtures/BeforeCleanupTest.groovy

```diff
@@ -27,9 +35,18 @@ class BeforeCleanupTest extends Specification implements YamlVirtualDirectory {
     def "should insert custom SQL if present"() {
         setup:
         def directory = unpackYamlToTempDirectory("custom.yml")
-
-        expect:
-        JFixturesOld.sql99(directory as String).asString() == CUSTOM_EXPECTED_SQL
+        def conf = "${directory}/.conf.yml";
+
+        when:
+        def sql = JFixtures
+            .ofConfig(conf)
+            .loadDirectory(directory as String)
```

Comment:
Is it possible to define string in a setup section?
`String directory = unpackYamlToTempDirectory("custom.yml")` and avoid casting here:
`.loadDirectory(directory)`

Reply:
We have a ticket already(#192) to overload `.loadDirectory` so it may consume `String`, `Path` and `File` as well and we'll have no need in this casting in all the tests

Reply:
:+1: